views/login/eligibility-age.pug

```diff
@@ -0,0 +1,18 @@
+extends ../base
+
+block variables
+  -var title = __('Age 65 and above')
+  -var ageYesNo = hasData(data, 'login.ageYesNo') ? data.login.ageYesNo : ''
```

Comment:
Looks like we're not actually using this variable down below (ie, it doesn't go anywhere).

stick it into the +radiosYesNo mixin as the `value=` so that the Yes/No is pre-checked when you reload the page for a question you've already answered.